checkout.go

```diff
@@ -87,14 +87,58 @@ type CheckoutApiResponse = ApiResponse[CheckoutAttributes, ApiResponseRelationsh
 // CheckoutsApiResponse is the api response for a list of checkout.
 type CheckoutsApiResponse = ApiResponseList[CheckoutAttributes, ApiResponseRelationshipsDiscount]
 
-// CheckoutCreateParams are parameters for creating a checkout
-type CheckoutCreateParams struct {
-	CustomPrice     int               `json:"custom_price"`
-	EnabledVariants []int             `json:"enabled_variants"`
-	ButtonColor     string            `json:"button_color"`
-	DiscountCode    *string           `json:"discount_code"`
-	CustomData      map[string]string `json:"custom_data"`
-	ExpiresAt       time.Time         `json:"expires_at"`
-	StoreID         string            `json:"store_id"`
-	VariantID       string            `json:"variant_id"`
+// Quantities represents variant quantities when creating checkout
+type Quantities struct {
```

Comment:
Let's use the same structure of variable names so this struct will be 

`CheckoutCreateDataVariantQuantity` instead of `Quantities`